api/organisations/subscriptions/licensing/helpers.py

```diff
@@ -0,0 +1,91 @@
+import base64
+import logging
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes, serialization
+from cryptography.hazmat.primitives.asymmetric import padding, rsa
+from django.conf import settings
+
+logger: logging.Logger = logging.getLogger(name=__name__)
+
+
+def sign_licence(licence: str) -> str:
+    message = licence.encode("utf-8")
+
+    # Load the private key from PEM
+    private_key = serialization.load_pem_private_key(
+        settings.SUBSCRIPTION_LICENCE_PRIVATE_KEY.encode("utf-8"), password=None
+    )
```

Comment:
Just for ergonomics, perhaps we should throw an exception (and catch it in the management command) if `SUBSCRIPTION_LICENCE_PRIVATE_KEY` is not set.

Reply:
Sounds good.